includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -190,8 +190,20 @@ public function show_messages() {
 	 */
 	public function get_action() {
 		$default_action = isset( $_SERVER['REQUEST_URI'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) : '';
+		$action = esc_url_raw( $this->action ? $this->action : $default_action );
 
-		return esc_url_raw( $this->action ? $this->action : $default_action );
+		/**
+		 * Alter form submit action URL
+		 *
+		 * Before submitting or editing a job, alter the posted values before they get stored into the database.
+		 *
+		 * @param string $action Form action value
```

Comment:
These need to be aligned.

Reply:
Updated and removed incorrect description, moved @since above params to match rest of plugin filters, added missing @return tag